src/App.jsx

```diff
@@ -130,9 +168,12 @@ function App() {
     };
     try {
       setIsSaving(true);
-      const resp = await fetch(url, options);
+      const resp = await fetch(
+        encodeUrl({ sortField, sortDirection, queryString }),
+        options
+      );
       if (!resp.ok) {
-        throw new Error('Failed completing todos');
+        throw new Error(`Error: ${resp.status} | Failed completing todos`);
       }
       const { records } = await resp.json();
       const completeTodo = {
```

Comment:
You are setting `isCompleted` to  `records[0].fields.isCompleted === true` why not just `records[0].fields.isCompleted`?

Also, you later set `isCompleted = true` if `records[0].fields.isCompleted`. What is the purpose of this? 

What is the goal of the logic in this try block? What is it currently doing?